SimpleSAMLphpAuth.php

```diff
@@ -0,0 +1,97 @@
+<?php
+# Licensed under the MIT license
+
+/**
+ * Sample Auth plugin
+ */
+class SimpleSAMLphpAuthPlugin extends MantisPlugin  {
+	/**
+	 * A method that populates the plugin information and minimum requirements.
+	 * @return void
+	 */
+	function register() {
+		$this->name = plugin_lang_get( 'title' );
+		$this->description = plugin_lang_get( 'description' );
+		$this->page = 'config_page';
+
+		$this->version = '0.1';
+		$this->requires = array(
+			'MantisCore' => '2.4.0',
+		);
+
+		$this->author = 'Erwann PENET';
+		$this->contact = '';
+		$this->url = 'https://github.com/mantisbt-plugins/SimpleSAMLphpAuth';
+	}
+
+	/**
+	 * plugin hooks
+	 * @return array
+	 */
+	function hooks() {
+		$t_hooks = array(
+			'EVENT_AUTH_USER_FLAGS' => 'auth_user_flags',
+		);
+
+		return $t_hooks;
+	}
+
+	function config() {
+		return array(
+			'autoloader_path'   => '',

```

Comment:
Why do we need a file path in the configs?  Why can't this path be known by the plugin assuming the library is bundled with the plugin?  Even if for some reason we think this is needed, I wouldn't surface it in the UI and keep it as an internal config option that can be overridden in `config/config_inc.php`.  But ideally, I would try to get rid of this config.

Reply:
This is linked to the workings of SimpleSAMLphp.
It is a standalone installation, separate from the core Mantis installation.